eZ Platform/Admin UI/USE_CASE.md

```diff
@@ -0,0 +1,25 @@
+# Work in Progress
+
+# Prototype: Hybrid Application
+## Goals:
+- Validate hypothesis of a hybrid application architecture.
+- Learn feasibility and difficulties
+
+## Stories
+
+### As a developer I want to simply reuse Admin Panel functionalities present in 1.x
+- ex: content type view
+
+### As a developer I want to simply reuse existing YUI-based modules present in 1.x
+- ex1: universal discovery widget
+- ex2: landing page editor
+- ex3: sub-item views in the repository
+
+### As a developer I want to extend the hybrid app with a React-based module.
```

Comment:
before thinking of anything based on a JavaSript framework, maybe we should first think about the case  of a custom module rendered server side to try to make super common usecase simple.